src/components/form/range/dual_range.test.tsx

```diff
@@ -217,36 +292,37 @@ describe('EuiDualRange', () => {
     // `ref`s to access the `onResize` instance method (search for `rangeRef.current?.onResize`)
     // If we switch EuiDualRange to a function component, we'll need to use `useImperativeHandle`
     // to allow Kibana to continue calling `onResize`
-    test('onResize', () => {
+    it('calls onResize', () => {
       // This super annoying type is now required to pass both the `ref` typing and account for instance methods
       type EuiDualRangeRef = React.ComponentProps<typeof EuiDualRange> &
         EuiDualRangeClass;
 
+      const onResize = jest.fn();
+
       const ConsumerDualRange = ({ callResize }: { callResize?: boolean }) => {
         const rangeRef = useRef<EuiDualRangeRef>(null);
 
         useEffect(() => {
           if (callResize) rangeRef.current?.onResize(500);
         }, [callResize]);
```

Comment:
See above comment - we can remove the `callResize` prop and rerender if we no longer need to spy on the component instance.

```suggestion
        useEffect(() => {
          rangeRef.current?.onResize(500);
        }, []);
```